source/common/filter/config_discovery_impl.cc

```diff
@@ -16,19 +16,28 @@
 namespace Envoy {
 namespace Filter {
 
-constexpr absl::string_view HttpStatPrefix = "http_filter.";
+const absl::string_view HttpStatPrefix = "http_filter.";
+const absl::string_view TcpListenerStatPrefix = "tcp_listener_filter.";
+const absl::string_view UdpListenerStatPrefix = "udp_listener_filter.";
```

Comment:
Why is this now defined as `const` and not `constexpr`?

Reply:
ACK